examples/tutorials/Advanced/Websockets - Federated training/README.md

```diff
@@ -0,0 +1,25 @@
+# Federated learning using websockets - MNIST example
```

Comment:
Could you make the name of this folder more command line friendly (ie no spaces

Reply:
Would the name "websockets-example-MNIST" be good?
Shall we change the "Advanced" folder to "advanced" to align it as well?